.github/codecov.yml

```diff
@@ -0,0 +1,2 @@
+comment:
+  layout: "reach,diff,footer"
```

Comment:
(very pedantic) Usually use single quotes for yaml

Reply:
```suggestion
  layout: 'reach,diff,footer'
```